Realm/ObjectServerTests/RLMObjectServerTests.m

```diff
@@ -20,6 +20,20 @@
 
 #import "RLMSyncUser+ObjectServerTests.h"
 
+static NSURL *makeRealmURL(const char *function, NSString *identifier) {
+    // 'function' is expected to be an Objective-C method name: "[MyClass fooBarBaz]"
```

Comment:
Any reason to not do something like:
```swift
var components = URLComponents(string: "realm://localhost:9080/~/")!
components.path += functionName
return components.url
```

That'll take care of escaping any characters in `functionName` that need to be escaped. It drops the whole `identifier` thing as that doesn't appear to be used.